perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -149,8 +160,24 @@ def _IsReady(self):
     Returns:
       True if the resource was ready in time, False if the wait timed out.
     """
+    cmd = util.GcloudCommand(self, 'sql', 'instances', 'describe',
+                             self.instance_id)
+    stdout, _, _ = cmd.Issue()
```

Comment:
Here and everywhere in this file where we try to do json.loads(stdout), it's quite fragile and needs to be wrapped inside try except and errors need to be handled. The reason being, stdout can be absent or does not contain a JSON, depending on the status of the command and such. We need to handle the error here and return false/true accordingly to the caller.